content/en/security/cspm/frameworks_and_benchmarks.md

```diff
@@ -36,19 +36,21 @@ Cloud Security Posture Management (CSPM) comes with more than 400 out-of-the-box
 
 *To pass the Monitoring Section of the [CIS AWS Foundations benchmark][2], you **must** enable [Cloud SIEM][11] and forward [Cloudtrail logs to Datadog][12].
 
-**Some [CIS Kubernetes Benchmark][5] detection rules only apply to self-hosted Kubernetes clusters.
+**Some [CIS Kubernetes Benchmark][5] compliance rules only apply to self-hosted Kubernetes clusters.
 
 Datadog also provides Essential Cloud Security Controls, a set of recommendations developed by Datadog internal security experts. Based on common cloud security risks we have observed at Datadog, this ruleset aims to help users new to cloud security easily remediate high-impact misconfigurations across their cloud environments.
 
-**Note**: CSPM provides visibility into whether your resources are configured in accordance with certain detection rules. These rules address various regulatory frameworks, benchmarks, and standards ("Security Posture Frameworks"). CSPM does not provide an assessment of your actual compliance with any Security Posture Framework, and the detection rules may not address all configuration settings that are relevant to a given framework. Datadog recommends that you use CSPM in consultation with your legal counsel or compliance experts.
+**Notes**:
+- CSPM provides visibility into whether your resources are configured in accordance with certain compliance rules. These rules address various regulatory frameworks, benchmarks, and standards (Security Posture Frameworks). CSPM does not provide an assessment of your actual compliance with any Security Posture Framework, and the compliance rules may not address all configuration settings that are relevant to a given framework. Datadog recommends that you use CSPM in consultation with your legal counsel or compliance experts.
+- The compliance rules for the CIS benchmarks follow the CIS automated recommendations. If you are obtaining CIS certification, we recommend reviewing the manual recommendations as part of your overall security assessment.
```

Comment:
@PronoyC I've had this PR open for a while. Do we still need to add this note to the CSPM compliance frameworks article?